### DISCUSSION TITLE
How do I get started, I'm still learning but want to begin.

### DISCUSSION THREAD

**Discussion by Pmspaethe** [twbs/bootstrap#40781] [General]
Like the title says lol 
Any help or advice
Thanks:)

**Comment by julien-deramond** [comment]
Hey @Pmspaethe 
From the documentation, you could start from our [Getting started](https://getbootstrap.com/docs/5.3/getting-started/introduction/) to put in place your project and understand what Bootstrap is about. Then, you could try to add [Components](https://getbootstrap.com/docs/5.3/components/) in your page, then organize them with some [Layout](https://getbootstrap.com/docs/5.3/layout/breakpoints/) to handle grids, columns, and breakpoints.
In order to help, you have some ready-to-reuse [Examples](https://getbootstrap.com/docs/5.3/examples/) that you can start from.
But also, there are a lot of articles and videos tutorials out there that explain the basics of Bootstrap too. I would use this kind of material to be guided :)
Have fun!
